website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -88,15 +88,6 @@ instance, and click **Authenticate**.
    instance's Global scope via the web UI. This is the default scope for all new
    Boundary clusters.
 
-1. Click **Auth Methods**.
-
-1. Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
-authenticate to HCP Boundary using the CLI.
-
-<Tabs>
-<Tab heading="CLI">
```

Comment:
I think these two missing tags might be causing the Vercel error:

https://vercel.com/hashicorp/boundary/58KYr2mkzLii6Y2UvxGLd7YghqYQ#L161